hs_core/views/resource_folder_hierarchy.py

```diff
@@ -71,6 +71,15 @@ def data_store_structure(request):
     res_coll = os.path.join(resource.root_path, store_path)
     try:
         store = istorage.listdir(res_coll)
+        dirs = []
+        for dname in store[0]:  # directories
+            d_pk = dname.decode('utf-8')
+            name_with_full_path = os.path.join(res_coll, d_pk)
+            # TODO /django_irods/download/ shouldn't be hardcoded
+            d_url = '/django_irods/download/zips/' + name_with_full_path
+            # TODO check if folder is found in django
```

Comment:
I'm just going to remove this comment unless someone knows a good way to check if the folder exists.  I was trying to follow the pattern set for files, but folder information isn't relayed to here.